.gitignore

```diff
@@ -27,6 +27,7 @@ msbuild.log
 msbuild.err
 msbuild.wrn
 msbuild.binlog
+msbuild.ProjectImports.zip
```

Comment:
Why doesn't this file land somewhere in a bin or obj folder and get automatically ignored?

Reply:
This can happen when you cancel a build process while running with `/bl`. This will go in whatever the current working directory is.